subsys/bluetooth/controller/hci/hci.c

```diff
@@ -1704,6 +1704,27 @@ static void vs_write_bd_addr(struct net_buf *buf, struct net_buf **evt)
 	ccst->status = 0x00;
 }
 
+static void vs_read_build_info(struct net_buf *buf, struct net_buf **evt)
+{
+	struct bt_hci_rp_vs_read_build_info *rp;
+
+#define BUILD_TIMESTAMP " BUILD: " __DATE__ " " __TIME__
+#define HCI_VS_BUILD_INFO "Zephyr OS v" \
+	KERNEL_VERSION_STRING BUILD_TIMESTAMP \
+	" (" CONFIG_BT_CTLR_HCI_VS_BUILD_INFO ")"
+
+	const char build_info[] = HCI_VS_BUILD_INFO;
+#define BUILD_INFO_EVT_LEN (sizeof(struct bt_hci_evt_hdr) + \
+			    sizeof(struct bt_hci_evt_cmd_complete) + \
+			    sizeof(struct bt_hci_rp_vs_read_build_info) + \
+			    sizeof(build_info))
+	BUILD_ASSERT(CONFIG_BT_RX_BUF_LEN > BUILD_INFO_EVT_LEN);
```

Comment:
Is > really correct here, shouldn't it be >=?

Reply:
Yep, will fix